lib/active_admin/csv_builder.rb

```diff
@@ -48,14 +48,20 @@ def build(controller, receiver)
         receiver << byte_order_mark
       end
 
+      options_for_csv = options.reject { |k, _v| k == :encoding_options }
+
       if options.delete(:column_names) { true }
-        receiver << CSV.generate_line(columns.map{ |c| encode c.name, options }, options)
+        receiver << CSV.generate_line(
+          columns.map { |c| encode c.name, options },
+          options_for_csv)
```

Comment:
It's cleaner to use [`except`](http://api.rubyonrails.org/classes/Hash.html#method-i-except):
```ruby
options_for_csv = options.except :encoding_options
```

But I'd prefer this code style:
```ruby
columns = columns.map{ |c| encode c.name, options }
receiver << CSV.generate_line(columns, options.except(:encoding_options))
```